toolkit/tools/pkg/imagecustomizerlib/customizeos.go

```diff
@@ -80,7 +80,7 @@ func doOsCustomizations(buildDir string, baseConfigPath string, config *imagecus
 		return err
 	}
 
-	verityUpdated, err := enableVerityPartition(config.OS.Verity, imageChroot)
+	verityUpdated, err := enableVerityPartition(config.Storage.Verity, imageChroot)
```

Comment:
Since the Verity config has been moved to `.storage`, does it still make sense to put it under the `customizeos`? Though I agree here is still the best sequence place.

Reply:
We are still making changes to the OS to prepare for verity. So, I think it makes sense.